Update/wata_012_02.txt

```diff
@@ -3842,7 +3848,7 @@ void main()
 //　一人に石を投げられたら、二人で石を投げ返しなさいと＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　一人に石を投げられたら、二人で石を投げ返しなさいと。",
-		   NULL, "If they throw a stone at one of you... then two of you you must throw stones back.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "\"If they throw a stone at one of you... then two of you must throw stones back.\"", GetGlobalFlag(GLinemodeSp));
```

Comment:
Not sure about the quotes.

Reply:
Makes it clearer it's someone saying it.